### ISSUE TITLE
Glob compatibility with latest stable release of vite@5.1.0

### ISSUE THREAD

**Issue by pdanpdan** [vikejs/vike#1485]
### Description

Hello.

Latest vite(@5.1.0) deprecated `as` glob option and moved its functionality to `query`.

```
The glob option "as" has been deprecated in favour of "query". Please update `as: 'extractExportNames'` to `query: '?extractExportNames'`.
The glob option "as" has been deprecated in favour of "query". Please update `as: 'extractAssets'` to `query: '?extractAssets'`.
```

**Comment by brillout** [comment]
It's already [fixed](https://github.com/vikejs/vike/commit/f7e5cd85d015cc08d3757b305b3f47f1c5f9e4e4) in `main`, which is pre-released in `0.4.161-commit-6a208ab`.

**Comment by brillout** [comment]
Actually, I just tried and I still see the warning 🤔 Let me dig.

**Comment by brillout** [comment]
Alright, it's fixed at `0.4.161-commit-32e5021`.

**Comment by Seapebble** [comment]
how do i fix this i got the error in my client
